src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const th = document.createElement('th');
+
+th.textContent = table.tHead.rows[0].cells[1].textContent;
```

Comment:
Check if `table.tHead` and `table.tHead.rows[0]` exist before accessing `cells[1]`. If they do not exist, this line will throw an error.